build/ensure-windows-toolchain.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+CC=x86_64-w64-mingw32-gcc
+PKG=gcc-mingw-w64-x86-64
+
+type -f ${CC} 2>/dev/null && exit 0
+
+apt-get update && \
```

Comment:
If we make the change suggested above (to leave the build-[platform] targets alone and introduce new ci-build-[platform]) targets, then this will probably be fine for a while. It's unlikely we'll run into issues in our CI environment because that's relatively stable.